### ISSUE TITLE
[FR] Touch Display support

### ISSUE THREAD

**Issue by CopterTec** [CopterTec/Marlin-Sovol_SV01#16]
Support for the original Sovol touch display.

**Comment by HotwiresFPV** [comment]
When will this be available?

**Comment by CopterTec** [comment]
It is only possible if Sovol publishes the source code.

**Comment by HotwiresFPV** [comment]
> It is only possible if Sovol publishes the source code.

Sovol just released the source code.

**Comment by CopterTec** [comment]
It would also possible by try and error. But two problems: we need the hardware and time.

**Comment by HotwiresFPV** [comment]
> It would also possible by try and error. But two problems: we need the hardware and time.

I can test. I have E3D V6 Titan Extruder with BL touch and Sovol Silent board. Touchscreen not installed but I have it.

**Comment by CopterTec** [comment]
You can download the source code and test different settings... We don't have time for that at the moment. Sorry.

**Comment by HotwiresFPV** [comment]
> You can download the source code and test different settings... We don't have time for that at the moment. Sorry.

Compiled the firmware with the source code and not working properly. I just hope you guys continue supporting SV01

**Comment by CopterTec** [comment]
Hello, did you copy the config files from /config/Sovol to /Marlin?

**Comment by HotwiresFPV** [comment]
Yes. I did

**Comment by CopterTec** [comment]
Ok. Please reinstall VSCode and Platformio... Most of the time it's a library problem.

**Comment by HotwiresFPV** [comment]
I installed VScode and Platform IO on 2 different computers Mac and Windows
same results using coptertec source code and Sovol source code.

**Comment by CopterTec** [comment]
You did something wrong... but i don't know what. This is the wrong page for compiler issues. Please check the Platformio or Marlin documentation and ask Sovol on Facebook.

**Comment by HotwiresFPV** [comment]
With coptertec 1.6.3 source code and config I can compile fine. It has
something to do with sovol source code.

**Comment by CopterTec** [comment]
Yeah they change to much stuff in an unprofessional way of coding. But this page is only for CopterTec firmware issues. Not for general Marlin issues or Sovol bugs. So please contact Sovol directly ;-).
